ngraph_bridge/ie_executable.h

```diff
@@ -42,9 +44,8 @@ class IE_Executable final : public Executable {
  private:
   InferenceEngine::CNNNetwork m_network;
   InferenceEngine::InferRequest m_infer_req;
-  // This map holds the parameters we insert for functions with no input
-  // parameters
-  unordered_map<string, shared_ptr<ngraph::runtime::Tensor>> m_hoisted_params;
+  TensorMap m_inputs;
```

Comment:
Add a comment on the purpose and content structure of this member

Reply:
Isn't this self-explanatory too? These are inputs and outputs to the executable.